hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/condition/ExternalConditionEval.java

```diff
@@ -37,6 +40,10 @@
 
     private static final long serialVersionUID = 1L;
 
+    private enum TypeValue {
+        DATA, EVENT
+    }
```

Comment:
Do we really need the new enum or to store the Event in a field?  It may be sufficient to just add the new constructor for Event?  You have a String value using Data or Event.

If you really want to keep it, perhaps change from 'TypeValue' to 'ValueType'.